internal/payloads/error_payload.go

```diff
@@ -47,7 +47,21 @@ func NewInvalidRequestError(ctx context.Context, err error) *ResponseError {
 		logger.Error().Msg(msg)
 	}
 	return &ResponseError{
-		HTTPStatusCode: 500,
+		HTTPStatusCode: 400,
+		Message:        msg,
+		RequestId:      ctxval.RequestId(ctx),
+		Err:            err,
+		Context:        ctx,
+	}
+}
+
+func PubkeyAlreadyExistsError(ctx context.Context, err error) *ResponseError {
+	msg := "pubkey with such name or fingerprint already exists for this account"
+	if logger := ctxval.Logger(ctx); logger != nil {
+		logger.Error().Msg(msg)
+	}
+	return &ResponseError{
+		HTTPStatusCode: 400,
```

Comment:
Rails uses [`422`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422) on validation errors, should we do that as well, I'm fine with whatever, but being it a concious choice, so @lzap is that choice of general 400 intentional, or you just didn't want 500?

Reply:
Nah I was just lazy, 422 it is.